.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly make sure to also add node_modules to the .gitignore file as per the requirement.